public-api/config/cdr_versions_prod.json

```diff
@@ -2,13 +2,24 @@
   {
     "cdrVersionId": 1,
     "name": "Synthetic Dataset v1",
-    "isDefault": true,
     "dataAccessLevel": 1,
     "bigqueryProject": "all-of-us-ehr-dev",
     "bigqueryDataset": "synthetic_cdr20180606",
     "creationTime": "2017-12-26 00:00:00Z",
     "releaseNumber": 1,
     "numParticipants": 946237,
     "publicDbName": "synth_p_2018q3_1"
+  },
+  {
+    "cdrVersionId": 2,
+    "isDefault": true,
+    "name": "All of Us Dataset v1",
+    "dataAccessLevel": 1,
+    "bigqueryProject": "aou-res-curation-prod",
+    "bigqueryDataset": "R2018Q4R1",
+    "creationTime": "2018-11-13 15:09:27Z",
+    "releaseNumber": 1,
+    "numParticipants": 116460,
```

Comment:
Workbench has 116157 here - is there a reason for the difference?

Reply:
Not sure, seemed to have been like this before too. 
https://console.cloud.google.com/bigquery?project=all-of-us-rw-prod&authuser=1&j=bq:US:bquxjob_6200136e_16959a89c38&page=queryresults

Reply:
Thanks, I remember asking about this actually. More context here: https://pmi-engteam.slack.com/archives/C5TB0DP63/p1543272631085200